### ISSUE TITLE
3D Quiver Plot

### ISSUE THREAD

**Issue by SermetCagan** [vispy/vispy#1969]
I am trying to do some phase plotting for my research and I have 7 variables which I create a meshgrid using numpy as `np.mgrid[0:1:10j,.....,[0:1:10j]` and I give those grid values to a function to use them for alignment of arrow points. After that I use matplotlib's quiver to plot 3D quiver plot as, `plt.quiver(x,y,z,u,v,w)`. But trying to do this much load of work with matplotlib is impossible in my case since I can not use GPU power with matplotlib but as far as I understand, vispy can easily handle the size of data I have. 

Is there any way to recreate this program using ArrowVisuals + GridMesh or any other method?

**Comment by djhoese** [comment]
Do you have a screenshot of what the expected output looks like? @kmuehlbauer semi-recently worked in a WindBarbVisual I think that would be almost the same by the sounds of it. I'm not sure off-hand what the parameters are for that and am going offline for a bit, but given the time zone difference wanted to draw Kai's attention to this.

**Comment by SermetCagan** [comment]
Yes, the expected output would look like around the ballpark of the below image. I checked the WindBarbVisual example but it is not the result I want, the closest one actually the arrow_quivers but I could not manage to put arrow positions on a 3d mesh grid using GridMesh.

<img width="352" alt="Screen Shot 2020-12-29 at 09 52 08" src="https://user-images.githubusercontent.com/11969628/103264453-c3587b80-49bb-11eb-9949-672164131e93.png">

**Comment by kmuehlbauer** [comment]
@SermetCagan Did you get along with this? It would be interesting to learn how you solved it, finally.
